LibreNMS/Validations/Php.php

```diff
@@ -63,6 +63,13 @@ private function checkExtensions(Validator $validator)
                 $validator->fail("Missing PHP extension: $extension", "Please install $extension");
             }
         }
+
+        $suggested_extensions = array('posix' => 'php-process');
+        foreach ($suggested_extensions as $extension => $packages) {
+            if (!extension_loaded($extension)) {
+                $validator->warn("Missing optional PHP extension: $extension", "It is suggested you install $packages or the one that matches your php version");
```

Comment:
Will this not just lead to people asking about install this and should we not just have it as part of the install docs?

Reply:
Yeah, probably would be worth adding it to the install docs.  No change is necessary on Ubuntu.

Reply:
As in it's already installed?

Reply:
As in for Debian/Ubuntu, it is part of php core.